generator-theia/src/common/abstract-generator.ts

```diff
@@ -51,7 +51,7 @@ export abstract class AbstractGenerator {
                 return `Promise.resolve(${invocation})`;
             }
             return invocation;
-        }).map(statement => `.then(function () { return ${statement}.then(load) })`);
+        }).map(statement => `    .then(function () { return ${statement}.then(load) })`);
```

Comment:
to fix indentation in the generated source files: `src-gen/frontend/main.js` and `src-gen/backend/main.js`